src/main/java/io/nats/client/NKey.java

```diff
@@ -225,7 +225,7 @@ static int crc16(byte[] bytes) {
         }
     }
 
-    static char[] base32Encode(final byte[] bytes) {
+    public static char[] base32Encode(final byte[] bytes) {
```

Comment:
I'm okay with moving this to a support class

Reply:
Moved to RandomUtils support class... since it seems to contain an assortment of utilities such as bytesToLong() and nextLong(Random, long). I suspect that class was originally intended for actual random number generation type of stuff, but bytesToLong() has nothing to do with random generation... so it seems reasonable.